### PR TITLE
honggfuzz 0.6

### PR DESCRIPTION
**Pull request by david-w-millar**
Added formula for google's general purpose fuzzer - [honggfuzz](https://google.github.io/honggfuzz/) for review.

### CODE REVIEW

**Comment by DomT4** [comment]
```
== /usr/local/Library/Formula/honggfuzz.rb ==
C:  2:  7: Put one space between the method name and the first argument.
C:  4:  6: Put one space between the method name and the first argument.
C:  5:  9: Put one space between the method name and the first argument.
C:  8: 11: Put one space between the method name and the first argument.

1 file inspected, 4 offenses detected
Error: 2 problems in 1 formula

==> audit problems
honggfuzz:
 * Description is too long. "name: desc" should be less than 80 characters.
Length is calculated as honggfuzz + desc. (currently 83)
 * Please remove the indefinite article "A" from the beginning of the description
```

File: Library/Formula/honggfuzz.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Honggfuzz < Formula
+  desc     "A general-purpose, easy-to-use fuzzer with interesting analysis options."
```
Comment (DomT4) [inline Library/Formula/honggfuzz.rb]: As the audit highlights, you've indented too hard here. Should be one space indent for each. You don't need to worry about keeping them in line with each other.

**Comment by david-w-millar** [comment]
Thanks for the feedback - I'll make the required changes.

**Comment by david-w-millar** [comment]
Truncated description, now using one space between methods and arguments.

File: Library/Formula/honggfuzz.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Honggfuzz < Formula
+  desc "General-purpose, easy-to-use fuzzer with analysis options"
+  homepage "https://google.github.io/honggfuzz/"
+  url "https://github.com/google/honggfuzz/archive/0.6.tar.gz"
+  sha256 "c95600c83b5b7836f3fdb1641565fbb07ccbf485446861c7dcb82db4172a0aa1"
+
+  def install
+    system      "make"
```
Comment (bfontaine) [inline Library/Formula/honggfuzz.rb]: Please use only one space here: `system "make"`.

File: Library/Formula/honggfuzz.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    bin.install "honggfuzz"
+  end
+
+  test do
+    system bin/"honggfuzz", "-h"
```
Comment (bfontaine) [inline Library/Formula/honggfuzz.rb]: Can’t we write a test that does something more substantial than e.g. `--help`/`--version`?

**Comment by DomT4** [comment]
Ping on Baptiste's comments.

**Comment by bfontaine** [comment]
Closing due to no response; feel free to open another pull-request addressing the comments above. Thanks!

**Pull request closed without merging**